truffle/test/Owned.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const Owned = artifacts.require('Owned');
+const truffleAssert = require('truffle-assertions');
+
+contract('Owned', (accounts) => {
+    // Testing getOwner function.
+    it('Owner should be ' + accounts[0], () => {
+        return Owned.new()
```

Comment:
I like to use a `beforeEach` to set up a fresh contract for each `it`.

Reply:
Also, never leave the `{from: who?` to chance.